### DISCUSSION TITLE
How to use datasharing and UDP simultaneously?

### DISCUSSION THREAD

**Discussion by wjhtinger** [eProsima/Fast-DDS#5389] [General]
Just in a host use datasharing, and between host use UDP, automatically switch , how?

**Comment by MiguelCompany** [comment]
This is the default behavior, as long as the [data-sharing requirements](https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/datasharing.html#constraints) are met.

**Reply by wjhtinger** [reply]
You mean, firstly I set the publisher with datasharing transport, and it will send two transport data(datasharing and UDP) simultaneously, when there has other host subscribe this topic?
